backend/db_liquibase_action.sh

```diff
@@ -22,7 +22,7 @@ case "${action}" in
     
   rollback)
     echo "Rolling back to tag: ${rollback_tag}"
-    gradle liquibaseRollback -PliquibaseCommandValue="${rollback_tag}"
+    gradle liquibaseRollback -PliquibaseCount="${rollback_tag}"
```

Comment:
I wonder if the parameter name needs to be something like `PliquibaseTag`

Reply:
OMG nice catch 😭 🙌  I misread the command as `liquibaseRollbackCount` -- changing this now!